src/OutboundServer/handlers.js

```diff
@@ -156,4 +186,7 @@ module.exports = {
     '/accounts': {
         post: postAccounts
     },
+    '/requestToPay': {
+        post: postRequestToPay
```

Comment:
@ksatya77 I think you have forgotten to include a PUT handler for the outbound transfer request model. If the model halts after resolving payee there is no way for the DFSP backend to tell the scheme adapter to proceed with the next stage.

Reply:
I am planning to add this logic as part of a separate PR